apps/prairielearn/src/server.js

```diff
@@ -2356,7 +2356,8 @@ if (esMain(import.meta) && config.startServer) {
 
         logger.verbose(`Connecting to ${pgConfig.user}@${pgConfig.host}:${pgConfig.database}`);
 
-        await sqldb.initAsync(pgConfig, idleErrorHandler);
+        // If devMode=true also set errorOnUnusedParameters
+        await sqldb.initAsync(pgConfig, idleErrorHandler, config.devMode);
```

Comment:
Rather than adding another unnamed parameter, let's extend the first parameter (the big config object) with support for a `errorOnUnusedParameters: true` property.

Reply:
I have a preference for named parameters too, but I didn't feel comfortable overloading the object passed to pg.pool. Two objects - one for pgConfig and one for our postgres library config?

Reply:
I'm very comfortable overloading that object. We're essentially wrapping `pg` to include extra functionality, so having our behavior be "the same as `pg` but with extra options" makes a lot of sense to me. If you _really really_ want to avoid passing extra properties down to `new pg.Pool`, you can `_.omit` it, but I really don't think that's necessary.